src/apiClient.js

```diff
@@ -2031,6 +2031,65 @@ class ApiClient {
         });
     }
 
+    uploadItemSubtitle(itemId, language, isForced, file) {
+        if (!itemId) {
+            throw new SyntaxError('Missing itemId');
+        }
+
+        if (!language) {
+            throw new SyntaxError('Missing language');
+        }
+
+        if (typeof isForced !== 'boolean') {
+            throw new TypeError('Parameter isForced must be a boolean.');
+        }
+
+        if (!file) {
+            throw new SyntaxError('File must be a subtitle file.');
+        }
+
+        const format = file.name.substring(file.name.lastIndexOf('.') + 1).toLowerCase();
+
+        if (!['sub', 'srt', 'vtt', 'ass', 'ssa'].includes(format)) {
```

Comment:
Hmm, I approved way too fast maybe.

Although they might be not that widely used, maybe we should include all the subtitle formats mentioned [here](https://en.m.wikipedia.org/wiki/Category:Subtitle_file_formats)?

It's non-blocking imho though.

Reply:
I think we don't support a bunch of them.

We definitely should allow VTT, TTML, and PGS, though.